assets/app/views/main.js

```diff
@@ -34,8 +37,13 @@ var AppView = Backbone.View.extend({
     $('.alert-container').html('');
   },
   setAlert: function (text) {
+    var newAlert = $('<div/>', {
+      class: 'usa-alert usa-alert-error',
+      role: 'alert'
```

Comment:
Yeah! I had forgotten my jQuery syntax and was googling when I came across it. Not sure when they added it, but it reads way nicer than super long strings.